Docker/README.md

```diff
@@ -1,14 +1,16 @@
 # Docker
 [Docker](https://docs.docker.com) is a platform for developers and sysadmins to develop, ship, and run applications. Docker lets you quickly assemble applications from components and eliminates the friction that can come when shipping code. Docker lets you get your code tested and deployed into production as fast as possible.
 
+With Docker, developers can build any app in any language using any toolchain. “Dockerized” apps are completely portable and can run anywhere - colleagues’ macOS and Windows laptops, QA servers running Ubuntu in the cloud, and production data center VMs running Red Hat.
+
 ## Docker for Mac
 Docker for Mac is the current release of Docker for macOS. Requirements:
 - Mac must be 2010 or new model with Intel's hardware support for memory management unit (MMU, virtualization, and Unrestricted mode)
 - OS X El Capitan 10.11 and newer releases are supported
```

Comment:
Officially El Cap. is required but I run docker on Yosemite (10.10.5), not sure if its worth mentioning that it is possible.

Personally I would leave it out and encourage people to upgrade (just like myself 😆 ).

Reply:
Yeah I also had this thought, El Capitan seems a bit too old nowadays. I'll push a change to remove this :)